audit/format.go

```diff
@@ -576,14 +576,18 @@ func NewTemporaryFormatter(format, prefix string) *AuditFormatter {
 //
 // See the documentation of the controlling option in FormatterConfig for more information on the purpose.
 func doElideListResponseData(data map[string]interface{}) {
-	for k, v := range data {
+	doElideListResponseDataWithCopy(data, data)
+}
+
+func doElideListResponseDataWithCopy(inputData map[string]interface{}, outputData map[string]interface{}) {
+	for k, v := range inputData {
 		if k == "keys" {
 			if vSlice, ok := v.([]string); ok {
-				data[k] = len(vSlice)
+				outputData[k] = float64(len(vSlice))
```

Comment:
Just for my own curiosity... I'm curious why you didn't leave this as `int` here?

Reply:
Ah, I guess JSON marshaling will marshal floats which contain integers to ints, so perhaps this isn't problematic except for direct testing.

Reply:
I'll take a look.  My initial reason for this change may no longer be valid.

Reply:
Hmm, I think you might be correct. I strangely get type issues if I remove this and/or the value in the tests. Interestingly though, the typing for `Unconventional other values in a list response are not touched` is left as an int, so I do wonder if there's an inconsistency lurking somewhere...

Reply:
the problem is that the tfw.hashExpectedValueForComparison()  test function doesn't handle the elided data types.  I've added a fix.  Hopefully that will be acceptable.